example/scalars/generated.go

```diff
@@ -203,7 +203,7 @@ func (ec *executionContext) _Query_search(ctx context.Context, field graphql.Col
 			return graphql.Null
 		}
 	} else {
-		var tmp interface{} = map[string]interface{}{"location": "37,144", "isBanned": false}
+		var tmp interface{} = map[string]interface{}{"isBanned": false, "location": "37,144"}
```

Comment:
Oh-oh! I think the map is making this non deterministic. Probably needs a sort first?